frontend/server/configs.ts

```diff
@@ -120,6 +120,7 @@ export function loadConfigs(argv: string[], env: ProcessEnv): UIConfigs {
      * e.g. a valid header value for default values can be like `accounts.google.com:user@gmail.com`.
      */
     KUBEFLOW_USERID_PREFIX = 'accounts.google.com:',
+    SERVER_NAMESPACE = 'kubeflow',
```

Comment:
Just a NIT picking about the name SERVER_NAMESPACE, are we referring to the frontend server? If that is the case, it might be helpful to be more specific: FRONTEND_SERVER_NAMESPACE, so we can differentiate from other servers. It will be great to apply to other places in this PR as well.

Reply:
yep it is, will make the change

Reply:
@zijianjoy addressed the change, I've changed the inner config parameter to [serverNamespace](https://github.com/kubeflow/pipelines/pull/11059/files#diff-34d43cd4bd3e8f92ac5cd08d44ed10bb3eb86786f89fab9e9aaf77282e013e58R269)  as it belongs to the ServerConfig object which captures frontend server configurations, so imo it would be redundant naming it something like `frontendServerConfig`. Let me know if this works.

Reply:
Thank you for the change!